mrtt-ui/src/data/mockOrganizations.js

```diff
@@ -6,13 +6,13 @@ const mockOrganizations = [
   { id: 3, name: 'Fake Org 3', isCurrentUserMember: true },
   { id: 6, name: 'Fake Org 4', isCurrentUserMember: false },
   { id: 5, name: 'Fake Org 5', isCurrentUserMember: false },
-  { id: 6, name: 'Fake Org 6', isCurrentUserMember: false },
   { id: 7, name: 'Fake Org 7', isCurrentUserMember: true },
   { id: 8, name: 'Fake Org 8', isCurrentUserMember: false },
   { id: 9, name: 'Fake Org 9', isCurrentUserMember: false },
   { id: 10, name: 'Fake Org 10', isCurrentUserMember: false },
   { id: 11, name: 'Fake Org 11', isCurrentUserMember: true },
-  { id: 12, name: 'Fake Org 12', isCurrentUserMember: false }
+  { id: 12, name: 'Fake Org 12', isCurrentUserMember: false },
+  { id: 13, name: 'Fake Org 13', isCurrentUserMember: false }
 ]
```

Comment:
this fixes the not unique key react error

Reply:
nice!